tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -814,7 +839,33 @@ def test_read_json_with_no_schema(session, mode):
         Row('{\n  "color": "Red",\n  "fruit": "Apple",\n  "size": "Large"\n}')
     ]
 
+    # test read multiple files in a directory
+    json_path = f"@{tmp_stage_only_json_file}"
+
+    df1 = get_reader(session, mode).option("PATTERN", ".*json.*").json(json_path)
+    res = df1.collect()
+    res.sort(key=lambda x: x[0])
+    assert res == [
+        Row(
+            '{\n  "CoLor": 10,\n  "new_field": true,\n  "new_fruit": "NewApple",\n  "new_size": 10\n}'
+        ),
+        Row('{\n  "color": "Red",\n  "fruit": "Apple",\n  "size": "Large"\n}'),
+        Row('{\n  "color": true,\n  "fruit": "Banana",\n  "size": "Small"\n}'),
+    ]
+
+    if not local_testing_mode:
+        # query_test
+        res = df1.where(sql_expr("$1:color") == "Red").collect()
```

Comment:
I feel like we could rewrite this to run in local testing, like `df1["$1"]["color"]`.

Reply:
good suggestion and it worked, updated